### PR TITLE
Fix/binding mdib version

### PR DESCRIPTION
**Pull request by deichmab-draeger**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the [changelog](../CHANGELOG.md) accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Review by leon1995 — CHANGES_REQUESTED** [review]

File: src/sdc11073/mdib/providermdibxtra.py [hunk 1]
```diff
@@ -27,61 +27,74 @@ class ProviderMdibMethods:
...
+                                           and d.NODETYPE == pm.LocationContextDescriptor]
+            if not child_location_descriptors:
+                descr_cls = mdib.data_model.get_descriptor_container_class(pm.LocationContextDescriptor)
+                descr_container = descr_cls(handle=uuid.uuid4().hex, parent_handle=system_context_descriptor.Handle)
+                descr_container.SafetyClassification = mdib.data_model.pm_types.SafetyClassification.INF
+                mdib.descriptions.add_object(descr_container)
 
     def ensure_patient_context_descriptor(self):
-        """Create PatientContextDescriptor if there is none in mdib."""
+        """Create a LocationContextDescriptor if there is none in mdib."""
```
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/mdib/providermdibxtra.py:51]: yes

File: pyproject.toml [hunk 2]
```diff
@@ -98,7 +98,7 @@ log_file_format = "%(asctime)s %(levelname)s (%(threadName)-10s) %(filename)s:%(
 log_file_date_format = "%Y-%m-%d %H:%M:%S:%f"
 
 [tool.ruff]
-extend-select = [# https://beta.ruff.rs/docs/settings/#extend-select
+lint.extend-select = [# https://beta.ruff.rs/docs/settings/#extend-select
```
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply pyproject.toml:101]: done

[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply pyproject.toml:101]: done

File: src/sdc11073/mdib/providermdibxtra.py [hunk 3]
```diff
@@ -27,61 +27,74 @@ class ProviderMdibMethods:
...
-            system_context_container = mdib.descriptions.NODETYPE.get_one(pm.SystemContextDescriptor)
-            descr_cls = mdib.data_model.get_descriptor_container_class(pm.LocationContextDescriptor)
-            descr_container = descr_cls(handle=uuid.uuid4().hex, parent_handle=system_context_container.Handle)
-            descr_container.SafetyClassification = mdib.data_model.pm_types.SafetyClassification.INF
-            mdib.descriptions.add_object(descr_container)
+        system_context_descriptors = mdib.descriptions.NODETYPE.get(pm.SystemContextDescriptor, [])
+        location_context_descriptors = mdib.descriptions.NODETYPE.get(pm.LocationContextDescriptor, [])
+
+        for system_context_descriptor in system_context_descriptors:
+            child_location_descriptors = [ d for d in location_context_descriptors
```
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/mdib/providermdibxtra.py:41]: done

**Pull request merged**